src/components/scrollbar-container/index.tsx

```diff
@@ -23,6 +23,11 @@ export class ScrollbarContainer extends React.Component<Properties, State> {
     this.checkScrollBottom = this.checkScrollBottom.bind(this);
   }
 
+  // used when we have "new" results while searching conversations, to scroll to the top
```

Comment:
Probably don't need this comment. Anything that renders with a scrollbar can use this if we want.